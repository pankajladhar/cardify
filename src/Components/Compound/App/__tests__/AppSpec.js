import React from 'react';

describe('App', () => {
    it('', () =>{
        expect(true).toBe(true)
    })
})