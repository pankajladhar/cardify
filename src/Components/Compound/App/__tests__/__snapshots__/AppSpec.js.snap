// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  className="Container"
>
  <header
    className="Header"
  >
    <a
      className="Header__Logo"
      href="/"
    >
      <img
        alt="Cardify"
        className="Header__Logo__Image"
        src="logo.svg"
      />
      Cardify
    </a>
  </header>
  <div
    className="Cards"
  >
    <div
      className="Card"
    >
      HDFC
       
      HDFC Card
    </div>
    <div
      className="Card"
    >
      CITI Bank
       
      CITI Bank Card
    </div>
    <div
      className="Card"
    >
      TATA SBI
       
      TATA SBI Card
    </div>
  </div>
  <footer
    className="Footer"
  >
    <ul>
      <li>
        <a
          href=""
        >
          Link 1
        </a>
      </li>
      <li>
        <a
          href=""
        >
          Link 1
        </a>
      </li>
      <li>
        <a
          href=""
        >
          Link 1
        </a>
      </li>
    </ul>
  </footer>
</div>
`;
